docs/source/membership/membership.md

```diff
@@ -1,341 +1,194 @@
+
+In order to use the Node OU roles, the "identity classification" feature must be enabled for the network. When using the folder-based MSP structure, this is accomplished by enabling "Node OUs" in the config.yaml file which resides in the root of the MSP folder:
+
+```
+NodeOUs:
+  Enable: true
+  ClientOUIdentifier:
+    Certificate: cacerts/ca.sampleorg-cert.pem
+    OrganizationalUnitIdentifier: client
+  PeerOUIdentifier:
+    Certificate: cacerts/ca.sampleorg-cert.pem
+    OrganizationalUnitIdentifier: peer
+  AdminOUIdentifier:
+    Certificate: cacerts/ca.sampleorg-cert.pem
+    OrganizationalUnitIdentifier: admin
+  OrdererOUIdentifier:
+    Certificate: cacerts/ca.sampleorg-cert.pem
+    OrganizationalUnitIdentifier: orderer
+```
+
+In the example above, there are 4 possible Node OU `ROLES` for the MSP:
+
+   * client
+   * peer
+   * admin
+   * orderer
+
+This convention allows you to distinguish MSP roles by the OU present in the CommonName attribute of the X509 certificate. The example above says that any certificate issued by cacerts/ca.sampleorg-cert.pem which has OU=client will identified as a client, OU=peer as a peer, etc. Starting with Fabric v1.4.3, there is also an OU for the orderer and for admins. The new admins role means that you no longer have to explicitly place certs in the admincerts folder of the MSP directory. Rather, the `admin` role present in the user's signcert qualifies the identity as an admin user.
+
+These Role and OU attributes are assigned to an identity when the Fabric-CA or SDK is used to `register` a user with the CA. It is the subsequent `enroll` user command that generates the certificates in the users's /msp folder.   
+
+![MSP1c](./ca-msp-visualization.png)
+
+The resulting ROLE and OU attributes are visible inside the X.509 signing certificate located in the /signcerts folder. The `ROLE` attribute is identified as `hf.Type` and  refers to an actor's role within its organization, (specifying, for example, that an actor is a `peer`). See the following snippet from a signing certificate shows how the Roles and OUs are represented in the certificate.
+
+![MSP1d](./signcert.png)
+
+Finally, though this is a slight misuse of OUs, they can sometimes be used by different organizations in a consortium to distinguish each other. In such cases, the different organizations use the same Root CAs and Intermediate CAs for their chain of trust, but assign the `OU` field to identify members of each organization.
```

Comment:
I'd like to see some clarification on how this is misue (or whether it actually is). I know this language comes from the previous version of the doc but still. If it's misuse why say it?

Reply:
I talked with Anthony to understand the origin of this comment and settled on this clarification:

Finally, OUs could be used by different organizations in a _consortium_ to distinguish each other. But in such cases, the different organizations have to use the same Root CAs and Intermediate CAs for their chain of trust, and assign the `OU` field to identify members of each organization. When every organization has the same CA or chain of trust, this makes the system more centralized than what might be desirable and therefore deserves careful consideration on a blockchain network.